src/main/kotlin/xyz/atrius/waystones/handler/WaystoneHandler.kt

```diff
@@ -79,5 +79,5 @@ class WaystoneHandler(
     }
 
     private fun distanceError(name: String, distance: Double, range: Int): String =
-        "$name is out of warp range [%.1f block(s)]".format(distance - range)
-}
\ No newline at end of file
+        localization.localize("warp-out-of-range", name, distance - range)
```

Comment:
A bit concerned about this line. Does this preserve the single-decimal rounding from the original code?

Reply:
Yes.